empyrion-galactic-survivalconfig.json

```diff
@@ -353,7 +353,7 @@
     "EnumValues": {}
   },
   {
-    "DisplayName": "Custom Scenario",
+    "DisplayName": "Scenario",
     "Category": "Emp - Game Settings",
     "Description": "Choose from premade scenarios.",
```

Comment:
Maybe note here that that name of the scenario needs to be set in the Custom Scenario setting?